opencti-platform/opencti-graphql/src/domain/feed.ts

```diff
@@ -36,6 +37,11 @@ const checkFeedIntegrity = (input: FeedAddInput) => {
       }
     });
   }
+
+  // our stix matching is currently limited, we need to validate the input filters
+  if (input.filters && input.filters !== '{}') {
```

Comment:
can input.filters be equal to {} ?  I think it's undefined or of type FilterGroup (mode is compulsory etc)

Reply:
it can be apparently. I tested it, but don't remember the actual test case.

Well in fact this check is not useful, the validation would throw an error properly anyway..